### PR TITLE
Ansible-Runner - Azure pip modules

### PR DESCRIPTION
**Pull request by SergeBakharev**
## Status
[x] Ready
[] In Progress
[ ] In Hold (Reason for hold)

## Related Content Pull Request
https://github.com/demisto/content/pull/12795

## Description
Required Azure modules to be used with Ansible 2.10

### CODE REVIEW

**Comment by SergeBakharev** [comment]
Install of `vsphere-automation-sdk` looks funny but this is the only way I could find to install it via pipfile. [Open issues](https://github.com/vmware/vsphere-automation-sdk-python/issues/38) exist. Installing this using pipenv via git tag fails because there are multiple wheels with cross dependencies within the same repo. Installing via a .gz release file however works.

@glicht  Ready for your review.

File: docker/ansible-runner/Pipfile [hunk 1]
```diff
@@ -6,23 +6,65 @@ verify_ssl = true
...
+azure-mgmt-iothub = "==0.7.0"
+azure-mgmt-recoveryservices = "==0.4.0"
+azure-mgmt-recoveryservicesbackup = "==0.6.0"
+azure-mgmt-eventhub = "==2.0.0"
+requests-ntlm = "==1.1.0"
+PyYAML = "==5.4.1"
+vsphere-automation-sdk = {file = "https://github.com/vmware/vsphere-automation-sdk-python/archive/v7.0.1.0.tar.gz"}
 
 [requires]
 python_version = "3.7"
```
Comment (glicht) [inline docker/ansible-runner/Pipfile:70]: @SergeBakharev  Let's change this to 3.9. You will need to regenerate the lock file after this change.

**Comment by glicht** [comment]
@SergeBakharev please review what packages actually require pinning. For example: `requests-ntlm` and `PyYAML` I don't think should be pinned.

**Comment by SergeBakharev** [comment]
I've revised the pipfile and pinned only the Azure and ansible modules, everything else can be latest compatible version.

**Comment by SergeBakharev** [comment]
@glicht Missing Licenses added to known_licenses.json, ready for your approval.

**Review by glicht — APPROVED** [review]

**Pull request merged**